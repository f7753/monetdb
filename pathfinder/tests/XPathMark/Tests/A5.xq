doc("document_2.xml")//xlink:*
