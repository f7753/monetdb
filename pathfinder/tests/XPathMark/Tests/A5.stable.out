stdout of test 'A5` in directory 'tests/XPathMark` itself:


# 19:14:00 >  
# 19:14:00 >  ( pf -M -O3 A5.xq | Mtimeout -timeout 60 Mserver "--config=/var/tmp/_Stable_PREFIX_MONETDB_GNU_32_d__/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/var/tmp/_Stable_PREFIX_PATHFINDER_GNU_32_d__/lib/MonetDB:/var/tmp/_Stable_PREFIX_MONETDB_GNU_32_d__/lib/MonetDB" --set "gdk_dbfarm=/var/tmp/_Stable_PREFIX_MONETDB_GNU_32_d__/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_Stable_PREFIX_MONETDB_GNU_32_d__/var/MonetDB/log" --set mapi_port=40231 --set sql_port=58305 --set monet_prompt= --dbname=mTests_tests_XPathMark   --dbinit='module(pathfinder,pf_support,ascii_io);' )
# 19:14:00 >  

# Monet Database Server V4.6.0_rc3
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for i686-pc-linux-gnu/32bit; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.
<?xml version="1.0" encoding="utf-8"?>
<XQueryResult xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svg="http://www.w3.org/2000/svg">
</XQueryResult>
